tests/http_layer/test_request_construction.py

```
@@ -261,11 +261,11 @@ async def test_empty_data_returns_empty_dict(self) -> None:
 
 
 class TestInfoToolRequests:
-    """Verify unraid_info tool constructs correct GraphQL queries."""
+    """Verify unraid system tool constructs correct GraphQL queries."""
 
     @staticmethod
     def _get_tool():
```

Ruff (0.15.5)

[warning] Missing return type annotation for staticmethod `_get_tool`

(ANN205)